src/components/CreateSecret/Form/Annotations.test.js

```diff
@@ -16,124 +16,94 @@ import { fireEvent, waitForElement } from 'react-testing-library';
 import { Provider } from 'react-redux';
 import configureStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
-import { renderWithIntl } from '../../utils/test';
-import SecretsModal from '.';
+import { renderWithIntl, rerenderWithIntl } from '../../../utils/test';
+import Annotations from './Annotations';
 
 const middleware = [thunk];
 const mockStore = configureStore(middleware);
 
-const secrets = {
-  byNamespace: {},
-  errorMessage: null,
-  isFetching: false
-};
-
-const namespaces = {
-  byName: {
-    default: {
-      metadata: {
-        name: 'default',
-        selfLink: '/api/v1/namespaces/default',
-        uid: '32b35d3b-6ce1-11e9-af21-025000000001',
-        resourceVersion: '4',
-        creationTimestamp: '2019-05-02T13:50:08Z'
-      }
-    }
-  },
-  errorMessage: null,
-  isFetching: false,
-  selected: 'default'
-};
-
-const serviceAccountsByNamespace = {
-  blue: {
-    'service-account-1': 'id-service-account-1',
-    'service-account-2': 'id-service-account-2'
-  },
-  green: {
-    'service-account-3': 'id-service-account-3'
-  }
-};
-
-const serviceAccountsById = {
-  'id-service-account-1': {
-    metadata: {
-      name: 'service-account-1',
-      namespace: 'blue',
-      uid: 'id-service-account-1'
-    }
-  },
-  'id-service-account-2': {
-    metadata: {
-      name: 'service-account-2',
-      namespace: 'blue',
-      uid: 'id-service-account-2'
-    }
-  },
-  'id-service-account-3': {
-    metadata: {
-      name: 'service-account-3',
-      namespace: 'green',
-      uid: 'id-service-account-3'
-    }
-  }
-};
-
-const store = mockStore({
-  secrets,
-  namespaces,
-  notifications: {},
-  serviceAccounts: {
-    byId: serviceAccountsById,
-    byNamespace: serviceAccountsByNamespace,
-    isFetching: false
-  }
-});
+const store = mockStore({});
 
 const gitServerPlaceholder = new RegExp('https://github.com', 'i');
-const dockerRegistryPlaceholder = new RegExp(
-  'https://index.docker.io/v1/',
-  'i'
-);
 
-it('Test SecretsModal Server URL examples', async () => {
-  const handleNew = jest.fn();
+it('Test CreateSecret Server URL examples', async () => {
```

Comment:
include the component name at the start of the test description or wrap the `it`s in a `describe` so they're easier to filter and give more helpful error messages in the failure summary

e.g.

```js
describe('Annotations', () => {
  it('should ...', () => {
    ...
  });
});
```

or

```js
it('Annotations should ...', () => {
  ...
});
```